Drop-In/src/main/java/com/braintreepayments/api/DropInAnalyticsEvent.java

```diff
@@ -0,0 +1,43 @@
+package com.braintreepayments.api;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+class DropInAnalyticsEvent implements Parcelable {
+
+    private final String fragment;
```

Comment:
Can we call this `eventFragment` or `eventString`? `fragment` alone is a little confusing as the name here when it's not an Android Fragment.

Reply:
I agree. Fragment is kind of legacy since that's what the parameter is named in core, since we tack on `android.dropin2` at somepoint I think.

Reply:
The idea was `event.getFragment()` would be less reduntant, maybe `event.getName()` woud be better?

Reply:
Ended up going with name.